plugins/CoreAdminHome/templates/generalSettings.twig

```diff
@@ -1,290 +1,226 @@
 {% extends 'admin.twig' %}
 
 {% block content %}
-{# load macros #}
-{% import 'macros.twig' as piwik %}
-{% import 'ajaxMacros.twig' as ajax %}
+    {% import 'macros.twig' as piwik %}
+    {% import 'ajaxMacros.twig' as ajax %}
 
-{% if isSuperUser %}
```

Comment:
only super users should be able to view these general settings, is is expected that this `if` statement was removed?